src/main/java/com/uber/cadence/internal/compatibility/proto/serviceclient/GrpcServiceStubs.java

```diff
@@ -124,7 +127,7 @@ final class GrpcServiceStubs implements IGrpcServiceStubs {
             MetadataUtils.newAttachHeadersInterceptor(headers),
             newOpenTelemetryInterceptor(),
             newOpenTracingInterceptor(options.getTracer()));
-    if (log.isTraceEnabled()) {
+    if (log.isTraceEnabled() || enableLogging) {
```

Comment:
chatgpt says you can change logger level in the code directly. If that's true, we may not need `enableLogging`
```
@Before
    public void setUp() {
        // Set the logging level to TRACE for detailed output in tests
        LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
        loggerContext.getLogger("ROOT").setLevel(Level.TRACE);
    }
```

Reply:
Done.